modules/reference/pages/data-transform-api.adoc

```diff
@@ -1,12 +1,9 @@
-= Data Transforms API
-:description: Working with data transformation APIs in Redpanda.
-
-{badge-tech-preview}
+= Golang Data Transforms API
```

Comment:
@rockwotj  how do we disambiguate Golang and Tinygo here , if at all. In other contexts you've used 'Golang' to mean 'big go'.  Should we just say 'Go'. Or is 'Golang' better here?  not sure

Reply:
Yes, Fastly chose Standard Go vs Tinygo

https://www.fastly.com/blog/announcing-standard-go-support-for-fastly-compute

Reply:
Also these are docs for the Golang SDK which is the same for both Standard Go and Tinygo so I don't think this needs to change.

Reply:
@rockwotj could you please confirm? Is it correct to leave this as Golang?

Reply:
yes please leave it.